bin/dlang/d2-release

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+# Copyright sociomantic labs GmbH 2017.
+# Distributed under the Boost Software License, Version 1.0.
+# (See accompanying file LICENSE.txt or copy at
+# http://www.boost.org/LICENSE_1_0.txt)
+set -eu
+
+# Paths
+r=$(dirname $0)/../..
+beaver=$r/bin/beaver
+
+# Use github utilities
+. $r/lib/github.sh
+
+# Convert the code to D2
+$beaver make DVER=2 d2conv
+
+set -x
+
+# Commit the changes and tag
+git config user.name="$(git for-each-ref --format="%(taggername)"
+        refs/tags/$TRAVIS_TAG)"
+git config user.email="$(git for-each-ref --format="%(taggeremail)"
+        refs/tags/$TRAVIS_TAG)"
+git commit -a -m 'Auto-convert to D2'
+
+# Create the new tag
+d2tag="$TRAVIS_TAG+d2"
+cat <<EOM | git tag -F- "$d2tag"
```

Comment:
TIL `-F` always implies `-a`

Reply:
Everything that adds a message to the tag, must imply `-a`, lightweight tags can't have messages :)